README.md

```diff
@@ -1,10 +1,48 @@
 [![Build Status](https://travis-ci.com/Java-Arctic-Ratel/repair-service.svg?branch=master)](https://travis-ci.com/Java-Arctic-Ratel/repair-service) [![codecov](https://codecov.io/gh/Java-Arctic-Ratel/repair-service/branch/master/graph/badge.svg)](https://codecov.io/gh/Java-Arctic-Ratel/repair-service)
-# repair-service
+# Repair service
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installing](#installing)
+* [Running test](#running-test)
 
+## General info
+Application for accounting and business automation in the field of electronic equipment repair services: a single database of orders and customers.
 
-## Requirements: 
-- JDK 11 or later
-- Maven 3+
-- PostgreSQL‍, H2 for dev
-- Docker
-- Spring Boot 2
\ No newline at end of file
+## Technologies
+- JDK11
+- Spring Boot 2
+- Gradle 5
+- Database for development environment:
+  - H2 (in memory)
+- Database for production environment:
+  - PostgreSQL 10
+  - MySQL 5
+
+## Installing
+To clone and run this application, you'll need:
+- Download project from github URL: 
+            
+      git clone https://github.com/Java-Arctic-Ratel/repair-service.git
+
+- Go into the repository: 
+        
+      cd repair-service
+
+- Build project
+
+      gradle clean build
+- Run applications with a database for development environment:
+  - H2 (in memory): 
+        
+        gradle bootRun --args='--spring.profiles.active=h2'
```

Comment:
For further improvements: I'm pretty confident that `dev` profile will have more specific configs than only another kind of DB. From my point of view it would be better to name this profile `dev` and it can include another profile `h2` with specific DB type